src/pykoala/corrections/throughput.py

```diff
@@ -17,18 +20,45 @@
 # Modular
 from pykoala import vprint
 from pykoala.corrections.correction import CorrectionBase
-from pykoala.rss import RSS
+from pykoala.data_container import RSS
 from pykoala import ancillary
 
 
 class Throughput(object):
+    """Class that represents a throughput data set.
+    
+    Attributes
+    ----------
+    throughput_data : np.ndarray
+        Array containing the values of the throughput.
+    throughput_error : np.ndarray
+        Array containing the associated error of ``throughput_data``.
+    throughput_path : str
+        Path to the original file that was used to initialise the throughput.
+    """
     def __init__(self, throughput_data, throughput_error,
                  throughput_file=None):
         self.throughput_data = throughput_data
         self.throughput_error = throughput_error
         self.throughput_file = throughput_file
 
     def to_fits(self, output_path=None):
+        """Save the current throughput into a FITS file.
+
+        Description
+        -----------
```

Comment:
No title, I think

Reply:
This is an optional title that by default is omitted in sphinx. I personally like it, but it won't make any difference in the web-based documentation.